kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/servlets/AjaxPublicationsListServlet.java

```diff
@@ -755,9 +741,8 @@ String displayDate(PublicationDetail pub, KmeliaSessionController kmeliaScc,
     }
   }
 
-  String displayFiles(PublicationDetail pub, boolean linkAttachment, boolean seeAlso, String userId,
-      String topicId, KmeliaSessionController kmeliaScc, MultiSilverpeasBundle resources)
-      throws IOException {
+  String displayFiles(PublicationDetail pub, boolean linkAttachment, boolean seeAlso,
+      KmeliaSessionController kmeliaScc, MultiSilverpeasBundle resources) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)